File: JAVA/Kode Program/Gpu.java
public class Gpu extends Processor {
    private int vramSize;
    private String architecture;

    public Gpu(int vramSize, String architecture, int coreCount, double clockSpeed, String merk, int harga) {
        super(coreCount, clockSpeed, merk, harga);
        this.vramSize = vramSize;
        this.architecture = architecture;
    }

    public void setVramSize(int vramSize) {
        this.vramSize = vramSize;
    }

    public int getVramSize() {
        return this.vramSize;
    }

    public void setArchitecture(String architecture) {
        this.architecture = architecture;
    }

    public String getArchitecture() {
        return this.architecture;
    }
}

File: PYTHON/Kode Program/Gpu.py
from Processor import Processor

class Gpu(Processor):
    def __init__(self, vram_size=0, architecture="", core_count=0, clock_speed=0.0, merk="", harga=0):
        super().__init__(core_count, clock_speed, merk, harga)
        self._vram_size = vram_size
        self._architecture = architecture

    def set_vram_size(self, vram_size):
        self._vram_size = vram_size

    def get_vram_size(self):
        return self._vram_size

    def set_architecture(self, architecture):
        self._architecture = architecture

    def get_architecture(self):
        return self._architecture